storagenode/src/main/java/io/coti/storagenode/database/DbConnectorService.java

```diff
@@ -106,24 +102,10 @@ private void addIndex(String indexName, String objectName, boolean fromColdStora
 
     @Override
     public ClusterGetSettingsResponse getClusterDetails(Set<String> indexes) throws IOException {
-        MainResponse mainResponse = restClient.info(RequestOptions.DEFAULT);
```

Comment:
i understand that this is not in use. but maybe its a pre-ready code that was supposed to be used for something... lets not delete it entirely